modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -55,6 +55,10 @@
     public static final String FILE_NAMING = "fileNaming";
     public static final String STRING_ENUMS = "stringEnums";
     public static final String STRING_ENUMS_DESC = "Generate string enums instead of objects for enum values.";
+    public static final String QUERY_PARAM_OBJECT_FORMAT = "queryParamObjectFormat";
+    public static final String QUERY_PARAM_OBJECT_FORMAT_DOT = "isQueryParamObjectFormatDot";
+    public static final String QUERY_PARAM_OBJECT_FORMAT_JSON = "isQueryParamObjectFormatJson";
+    public static final String QUERY_PARAM_OBJECT_FORMAT_KEY = "isQueryParamObjectFormatKey";
```

Comment:
are these 3 constants used?

Reply:
Yes, open to an alternative if there is one for passing into mustache template:

https://github.com/OpenAPITools/openapi-generator/pull/5790/files#diff-f61af8a4c4eb2720bfb15259e0dca419R170-R187

Reply:
I mean the actual java constants: `QUERY_PARAM_OBJECT_FORMAT_KEY`, `QUERY_PARAM_OBJECT_FORMAT_JSON`, `QUERY_PARAM_OBJECT_FORMAT_DOT` - they are defined here, but I can't see any code that refers to them. Am I missing the point?

Reply:
I think it's that I don't understand Java, I was just muddling through.